packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.html

```diff
@@ -0,0 +1,114 @@
+<div class="telecom-pack-xdsl-meetings">
+    <header class="widget-presentation-header">
+        <h2
+            class="widget-presentation-title"
+            data-translate="xdsl_meeting_slots"
+        ></h2>
+    </header>
+
+    <!-- LOADER -->
+    <div class="text-center" data-ng-if="$ctrl.loading">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <oui-message type="error" data-ng-if="$ctrl.errorMessage">
+        <span data-ng-bind-html="$ctrl.errorMessage"></span>
+    </oui-message>
+
+    <oui-message type="success" data-ng-if="$ctrl.successMessage">
+        <span data-ng-bind-html="$ctrl.successMessage"></span>
+    </oui-message>
+
+    <div
+        class="widget-presentation py-3 move-number-config ft-meeting"
+        data-ng-if="$ctrl.showMeetingSlots && !ctrl.loading && !$ctrl.successMessage"
+    >
+        <div class="row">
+            <div
+                class="font-weight-bold"
+                data-translate="xdsl_meeting_description"
+            ></div>
+
+            <div
+                data-ng-show="$ctrl.meetingSlots.canBookFakeMeeting && $ctrl.meetingSlots.slots.length"
+            >
+                <div class="input-group mt-3 col-md-12">
```

Comment:
**question:** Using `.row` (L26) with a `.col-md-12` children look weird.

Can you simplify the structure here by using a single `<div>` element instead (if possible)?